tests/exporters/onnx/test_exporters_onnx_cli.py

```diff
@@ -134,6 +135,31 @@ def test_all_models_tested(self):
         if len(missing_models_set) > 0:
             self.fail(f"Not testing all models. Missing models: {missing_models_set}")
 
+    @parameterized.expand(PYTORCH_STABLE_DIFFUSION_MODEL.items())
+    @require_torch
+    @require_vision
+    @require_diffusers
+    def test_exporters_cli_pytorch_cpu_stable_diffusion(self, model_type: str, model_name: str):
+        self._onnx_export(model_name, model_type)
+
+    @parameterized.expand(PYTORCH_STABLE_DIFFUSION_MODEL.items())
+    @require_torch_gpu
+    @require_vision
+    @require_diffusers
+    @slow
+    @pytest.mark.run_slow
```

Comment:
Why?
Maybe I forgot the reason for this.

Reply:
triggered everyday by [`test_exporters_slow.yml`](https://github.com/huggingface/optimum/blob/c4750f6014d8ac39bca780e1e4381b15eaab9197/.github/workflows/test_exporters_slow.yml) (not all tests are triggered for every PR)